core/src/main/java/com/bankbazaar/core/model/TripActivityCompositeKey.java

```diff
@@ -0,0 +1,16 @@
+package com.bankbazaar.core.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class TripActivityCompositeKey implements Serializable {
+    private Long tripId;
+    private Long activityId;
```

Comment:
Why is this required to be a composite key?

Reply:
I have removed this.